packages/firebase_auth/firebase_auth_platform_interface/lib/src/user_info.dart

```diff
@@ -20,8 +20,8 @@ class UserInfo {
           displayName: data['displayName'] as String?,
           photoUrl: data['photoUrl'] as String?,
           phoneNumber: data['phoneNumber'] as String?,
-          isAnonymous: data['isAnonymous'] as bool,
-          isEmailVerified: data['isEmailVerified'] as bool,
+          isAnonymous: data['isAnonymous'] as bool? ?? false,
+          isEmailVerified: data['isEmailVerified'] as bool? ?? false,
```

Comment:
The UserInfo class is used in providerData where the data is null because `isAnonymous` is always false but `isEmailVerified` is always true

What kind of code triggers an issue in your app?